jupiterone/questions/questions.yaml

```
@@ -2954,3 +2954,69 @@ questions:
         version: v1.3.1
         requirements:
           - '9.10'
+  - id: integration-question-azure-are-keyvault-secrets-older-than-60-days
+    title: Which of my Key Vault Secets are older than 60 days?
+    description: Identify which Key Vault Secrets are older than 60 days?
+    queries:
+      - name: Informative
+        query: |
+          FIND azure_keyvault_secret WITH updatedOn < Date.now - 60 days AS ks
+          RETURN ks.displayName, ks.enabled, ks.updatedOn, ks.createdOn, ks.expiresOn
+    tags:
+      - azure
+      - data
+  - id: integration-question-azure-who-has-access-to-key-vault-secret
+    title: Who has access to a Key Vault Secret?
+    description: Identify who has access to Key Vault Secrets
+    queries:
+      - name: Informative
+        query: |
+          FIND azure_keyvault_secret WITH enabled=true AS secret
+            THAT CONTAINS azure_keyvault_service AS service
+            THAT ALLOWS * AS x
+          RETURN secret.displayName, service.displayName, x.displayName
+    tags:
+      - azure
+      - access
+    - id: integration-question-azure-storage-accounts-that-allow-access-to-internet
```

has an extra space of indentaiton
```suggestion
   - id: integration-question-azure-storage-accounts-that-allow-access-to-internet
```